### PR TITLE
CI: add CI helper-based travis configuration

### PR DESCRIPTION
**Pull request by klauer**
CI is not currently enabled for this repository, so let's try the new shared configurations.

### CODE REVIEW

File: L2SIVacuum/POUs/Functions/Gauges/FB_MKS_937B.TcPOU [hunk 1]
```diff
@@ -13,7 +13,7 @@ VAR_INPUT
 	PG	:	ST_VG := g_DummyVG; // Gauge Structure used to Interlock the ion gauges
 END_VAR
 VAR_OUTPUT
-	{attribute 'pytmc' := pv: '}
+	{attribute 'pytmc' := 'pv: '}
```
Comment (ZLLentz) [inline L2SIVacuum/POUs/Functions/Gauges/FB_MKS_937B.TcPOU:16]: :+1: for linter checks

**Comment by ZLLentz** [comment]
It does not seem like the xfail is working

File: L2SIVacuum/POUs/Functions/Gauges/FB_MKS_937B.TcPOU (hunk 1 above)
Comment (ghalym) [reply L2SIVacuum/POUs/Functions/Gauges/FB_MKS_937B.TcPOU:16]: Thanks

**Review by ghalym — APPROVED** [review]

**Comment by klauer** [comment]
Yeah ☹️  not sure what to do about the xfail failing...

Edit: well, I guess this is a good opportunity to try importing individual tests. Apologies in advance for commit spam as I try this out.

**Review by ghalym — APPROVED** [review]

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**